br/pkg/lightning/mydump/region.go

```diff
@@ -268,7 +268,7 @@ func makeSourceFileRegion(
 	}
 	// If a csv file is overlarge, we need to split it into multiple regions.
 	// Note: We can only split a csv file whose format is strict.
-	if isCsvFile && dataFileSize > int64(cfg.Mydumper.MaxRegionSize) && cfg.Mydumper.StrictFormat {
+	if isCsvFile && cfg.Mydumper.StrictFormat && dataFileSize > int64(cfg.Mydumper.MaxRegionSize)*11/10 {
```

Comment:
What happens if the file size if slightly bigger the `int64(cfg.Mydumper.MaxRegionSize)*11/10`? 🤣

Reply:
Then the file will be split by `cfg.Mydumper.MaxRegionSize`, so the second chunk size is about 1/10 * cfg.Mydumper.MaxRegionSize.

Reply:
If the file size is slightly bigger the `int64(cfg.Mydumper.MaxRegionSize)* 2`, the third chunk size is very small, will this be a problem?

Reply:
Not a big problem. The common case is that the data export tool (like dumpling or mydumper) set the exported file size with `cfg.Mydumper.MaxRegionSize`, but the output file size might be slightly bigger or smaller， so we can avoid split a lot of small chunks.

Reply:
make that 11/10 a named constant...

Reply:
It will make the code a bit ugly because 11/10 is a float. I add a code comment to explain why the threshold need to be increased

Reply:
you can make "10" a constant and set the upper limit to `MaxRegionSize + MaxRegionSize/10`.